src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule72thesummaryfragment/InputCorrectSummaryJavaDocCheck.java

```diff
@@ -3,23 +3,18 @@
 /**
  * Some Javadoc A {@code Foo} is a simple Javadoc.
  */
-class InputCorrectSummaryJavaDocCheck {
```

Comment:
Please revert all superfluous changes to this input. It is making it hard to to understand what was changed.
Any new cases you have to add, please add them to the end or in a separate file.